### PR TITLE
TINY-6268: Updated description of images_upload_handler.

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket: 

Description of Changes:
Added the description for image_upload_handler

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by ashfordneil — APPROVED** [review]

File: _includes/configuration/images-upload-handler.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ The upload handler function takes four arguments:
 
 * `blobInfo`
 * A `success` callback
-* A `failure` callback
+* A `failure` callback that takes an error-message and an optional object containing:
```
Comment (metricjs) [inline _includes/configuration/images-upload-handler.md:9]: Not sure if "error-message" needs the hyphen - up to @tylerkelly13

File: _includes/configuration/images-upload-handler.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ The upload handler function takes four arguments:
...
+    * `remove` Removes the image from the document, defaults to `false`
```
Comment (metricjs) [inline _includes/configuration/images-upload-handler.md:10]:
```suggestion
    * `remove` - Removes the image from the document, defaults to `false`
```

Comment (metricjs) [reply _includes/configuration/images-upload-handler.md:10]: Or however @tylerkelly13 does these things; should maybe be a `:` instead?

**Review by metricjs — APPROVED** [review]

File: _includes/configuration/images-upload-handler.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/configuration/images-upload-handler.md:9]:
```suggestion
* A `failure` callback that takes an error message and an optional object containing:
```

File: _includes/configuration/images-upload-handler.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/configuration/images-upload-handler.md:10]: either's fine :)

**Review by tylerkelly13 — CHANGES_REQUESTED** [review]

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**